mmv1/third_party/terraform/tests/resource_sql_database_instance_test.go

```diff
@@ -1036,6 +1036,34 @@ func TestAccSqlDatabaseInstance_cloneWithDatabaseNames(t *testing.T) {
 	})
 }
 
+func TestAccSqlDatabaseInstance_cloneWithPreferredZone(t *testing.T) {
+	// Sqladmin client
+	acctest.SkipIfVcr(t)
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix":    acctest.RandString(t, 10),
+		"original_db_name": acctest.BootstrapSharedSQLInstanceBackupRun(t),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccSqlDatabaseInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
```

Comment:
Can add more steps here to test the updating of the field `preferred_zone`?

Reply:
Can you please add more test steps to update it from one zone to another zone? Thanks.

Reply:
The use case is first clone the instance to one zone, and then during the update, clone this instance to another zone.  Is it allowed to clone the instance to another zone during the update?

Reply:
Got you point, since preferred zone is not P0, dropping it from this PR, will follow up later on that. So it won't affect timely merging of other changes in this PR